src/tests/get_test_evidence_data.py

```
@@ -0,0 +1,132 @@
+
+    Returns:
+        List[str]: A list of page_ids.
+    """
+    page_ids = set()
+    for group in claim.get('evidence', []):
+        for item in group:
+            if isinstance(item, list) and len(item) > 0:
+                page_ids.add(str(item[0]))
+    return list(page_ids)
+
+def attach_wiki_content_to_claims(claims: List[Dict], wiki_index: dict) -> List[Dict]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)